runtime/descriptor.go

```diff
@@ -117,5 +119,31 @@ func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object
 		}
 		return ret, raised
 	}
+	if writtable {
+		setterFunc := func(f *Frame, args Args, _ KWArgs) (*Object, *BaseException) {
+			ret := None
+			var raised *BaseException
+			if raised = checkFunctionArgs(f, fieldName, args, ObjectType, ObjectType); raised == nil {
```

Comment:
Ah. Thanks for being consistent. I'm not sure why that code is written that way. It'd be great to clean it up as well. Thanks!